### ISSUE TITLE
Static Type Hinting: `dataprofiler/reports/graphs.py`

### ISSUE THREAD

**Issue by taylorfturner** [capitalone/DataProfiler#607]
**Is your feature request related to a problem? Please describe.**
Improve the project in the area of 
- Make collaboration and code contribution easier
- Getting started with the code when a contributor is new to the code base

**Describe the outcome you'd like:**
Utilizing types from [mypy](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html) documentation for python3, implement type hinting in `dataprofiler/reports/graphs.py`.

**Additional context:**
- [mypy](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html) documentation
- [Why use python types?](https://medium.com/vacatronics/why-you-should-consider-python-type-hints-770e5cb1570f)

**Comment by stefanycoimbra** [comment]
I'm working on this.

**Comment by taylorfturner** [comment]
Sounds great -- we will keep an eye out for the PR, @stefanycoimbra

**Comment by stefanycoimbra** [comment]
Yes, I'm trying to submit it now. Sorry for the delay, I had to fix my computer.

**Comment by taylorfturner** [comment]
Sounds good! I'll take a look tomorrow - thanks for the contribution!

**Comment by taylorfturner** [comment]
PR merged! Thanks again, @stefanycoimbra !
